lib/finch.ex

```diff
@@ -18,8 +18,18 @@ defmodule Finch do
   @pool_config_schema [
     protocol: [
       type: {:in, [:http2, :http1]},
+      deprecated: "Use :protocols instead."
+    ],
+    protocols: [
+      type:
+        {:in,
+         [
+           [:http1],
+           [:http2],
+           [:http1, :http2]
```

Comment:
It could also be `[:http2, :http1]`. Do we care? I would keep it as a list maybe and validate it later?